openshift/e2e-tests-openshift.sh

```diff
@@ -13,10 +13,15 @@ failed=0
 # Temp disabled due to running into OOM
 # (( !failed )) && run_unit_tests || failed=1
```

Comment:
Can you enable unit tests also? It seems I've missed it in my PR to master. Those shouldn't be problematic anymore.

Reply:
enabled